yt/testing.py

```diff
@@ -467,14 +471,13 @@ def construct_octree_mask(prng=RandomState(0x1d3d3d3), refined=[True]):
             construct_octree_mask(prng, refined)
     return refined
 
-def fake_octree_ds(prng=RandomState(0x1d3d3d3), refined=[True], quantities=None,
+def fake_octree_ds(prng=RandomState(0x1d3d3d3), quantities=None,
```

Comment:
Removing `refined` parameter would always create the octree mask with `refined=True`. I think we can keep `refined=None`, in this way we can also create the octree mask with `refined=False` flag.